src/cloud/docker/docker-development.md

```diff
@@ -40,7 +40,23 @@ You can use the Docker environment to emulate the {{site.data.var.ece}} Integrat
 -  [Docker architecture and common commands]({{site.baseurl}}/cloud/docker/docker-containers.html)
 -  [Launch Docker development environment]({{site.baseurl}}/cloud/docker/docker-config.html)
 
+## Getting support for {{site.data.var.mcd-prod}}
+
+{{site.data.var.mcd-prod}} is a Magento Community Engineering project supported by the Magento developer community. You have several options to get support and learn more about {{site.data.var.mcd-prod}} and Magento local development.
```

Comment:
It says Magento local development, but does it need to specify Cloud specifically, or is it okay to stay generic?

Reply:
Since you can use Docker for Cloud or on-premise Magento, I intentionally made it generic.